R/ttestonesample.R

```diff
@@ -104,10 +104,10 @@ TTestOneSample <- function(jaspResults, dataset = NULL, options, ...) {
       tzNote <- wNote <- NULL
       
       if (optionsList$wantsStudents || optionsList$wantsZtest)
-        tzNote <- gettextf("For the %s, location parameter is given by mean difference <em>d</em>.", testInNote)
+        tzNote <- gettextf("For the %s, location estimate is given by the sample mean <em>d</em>.", testInNote)
```

Comment:
This is less clear I think. Right now, JASP computes `mean(data - testValue)`. But the sample mean is independent of the test value. Also, the Descriptives table already shows the sample mean. So I think this footnote should explicitly mention that this is the difference between the sample mean and the test value (both here and in the documentation). Following that, I'm also not sure whether `Sample mean` is the right column title. We could also just stick to `Location Parameter`, because the changing column title can be somewhat confusing.

Reply:
I've changed the code now it shows the sample mean and not the mean difference. This was to fix the problem with the moving confidence intervals. I agree that the sample mean is shown multiple times. 

I could revert back to mean differences, but that'll give us the previous problem.